server/routes/user.js

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+import Sessions from '../controllers/Sessions';
+
+const { create } = Sessions;
+
+const router = express.Router();
+
+router.post('/users/login', create);
```

Comment:
it should be `router.post('/sessions/create', Sessions.create)`
remove the destructing, cos we could possibly have other create methods, like User.create

Reply:
Done